src/resources/js/tribe-common.js

```diff
@@ -85,6 +86,99 @@ tribe_auto_sysinfo.ajax = {
 
 	};
 
+	/**
+	 * Sets up listeners and callbacks to handle navigation to page #elements
+	 * gracefully and in a way that doesn't result in the admin toolbar obscuring
+	 * the target.
+	 */
+	my.navigate_to_id = function() {
+		$( document ).ready( my.maybe_navigate_to_id_on_doc_ready );
+		$( '.tribe_events_page_tribe-common' ).click( my.maybe_navigate_to_id_after_click );
```

Comment:
Perhaps alter this to the following to bind to dynamically added elements (if they are added in the future):

```js
$( document ).on( 'click', '.tribe_events_page_tribe-common', my.maybe_navigate_to_id_after_click );
```